.github/workflows/ci-pr.yml

```diff
@@ -27,11 +27,6 @@ jobs:
       with:
         maven-version: 3.8.8
         maven-mirror: 'https://dlcdn.apache.org/maven/maven-3/'
-    - name: Login dockerhub
```

Comment:
Why the removal?

Reply:
unlike other workflows it didn't seem to have access to the docker secrets and kept failing docker login.